### DISCUSSION TITLE
Until when does my Nuxt module need to support/be compatible with Nuxt 2? (Vue 2 extended support)

### DISCUSSION THREAD

**Discussion by dword-design** [nuxt/nuxt#24685] [Questions]
Hi guys, I've contributed several Nuxt modules and usually also provide Nuxt 2 compatibility. According to the docs, Nuxt 2 is maintained [until June 30, 2024](https://v2.nuxt.com/lts/). However, there will is also the [never-ending support for Vue 2](https://www.herodevs.com/support/nes-vue), so I was asking myself if I then still need to support Nuxt 2 after June 30.

Thanks already for help!

**Comment by danielroe** [comment]
It's totally up to you, as module author. I would think it's reasonable not to continue to maintain them after Nuxt 2 EOL.

**Comment by dword-design** [comment]
@danielroe Ok thanks that helps already. I was thinking about the [Extended Support](https://v2.nuxt.com/lts/) / [Never-Ending Support](https://www.herodevs.com/support/nes-vue) actually, so if authors should keep that in mind to support that?
